bagit/reader.go

```diff
@@ -128,6 +128,23 @@ func (r *Reader) loadtagfile(name string) error {
 	return scanner.Err()
 }
 
+// Checksum returns a pointer to the Checksum struct of the given filername
+// culled from the manifests map
+// If one does not exist for the given file, it returns nil
+
```

Comment:
remove this blank line. the godoc tool will ignore comments which are separated from the function by blank lines.